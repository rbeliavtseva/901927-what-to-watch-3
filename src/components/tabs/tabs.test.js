import React from "react";
import renderer from "react-test-renderer";
import Tabs from "../tabs/tabs.jsx";

it(`Render Tabs`, () => {
  const tree = renderer
    .create(
        <Tabs
          key={0}
          onClick={() => {}}
          activeIndex={0}
        >
          <div>
            Child1
          </div>
          <div>
            Child2
          </div>
        </Tabs>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
