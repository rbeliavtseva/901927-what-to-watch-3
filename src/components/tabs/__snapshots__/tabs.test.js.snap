// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Tabs 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item movie-nav__item--active"
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item "
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
        >
          Details
        </a>
      </li>
    </ul>
  </nav>
  <div>
    Child1
  </div>
</div>
`;
